crates/matrix-sdk-crypto/src/store/memorystore.rs

```diff
@@ -485,6 +465,26 @@ impl CryptoStore for MemoryStore {
         Ok(RoomKeyCounts { total, backed_up })
     }
 
+    async fn get_inbound_group_sessions_by_room_id(
+        &self,
+        room_id: &RoomId,
+    ) -> Result<Vec<InboundGroupSession>> {
+        let inbounds: Vec<InboundGroupSession> = self
+            .inbound_group_sessions
+            .read()
+            .get(room_id)
+            .expect("No inbound group sessions for that roomId")
```

Comment:
Panicking in rust should happen only in situations that you strongly believe should not happen. Having no sessions for a given room is a perfectly normal situation.